lib/net.js

```diff
@@ -234,9 +238,12 @@ function Socket(options) {
 }
 util.inherits(Socket, stream.Duplex);
 
+// Refresh existing timeouts.
```

Comment:
Ah, thanks – I think the comment is fine, maybe `_unrefActive()` just isn’t a very descriptive name.